### ISSUE TITLE
New Membership

### ISSUE THREAD

**Issue by geladariia** [anytypeio/roadmap#72]
**High level:**
A new membership model allows you to pay only for what you use, unlike the current bundled approach. More pricing options will be available. More complex use cases, such as those for businesses and communities, will also be addressed.

**Comment by robotflaqd** [comment]
I like what was said here [community.anytype.io/t/membership-plan-between-explorer-and-builder](https://community.anytype.io/t/membership-plan-between-explorer-and-builder/24848/2) about using the price reasonable for people in other countries. I personally think for US pricing something like a $5 or $6/month ($4 or $5/month if paid annually) would be a good middle between the $99 option; if it's reasonable for Anytype to meet it's financial metrics. That's for people in the US and first world countries. I also agree to have pricing match the state of the country, like most developing countries will have pricing match the currency exchange rate like what Steam does with their games, just like what's said in that post. 

What that commenter left out in that example is that Notesnook operates very lean, I believe with a very small team. They also don't have VC funding so their financial situation is different from Anytype. 

I mentioned in [another comment](https://github.com/anyproto/roadmap/issues/8#issuecomment-2376559721) here in this roadmap that I plan to use Anytype for life, and I believe pricing would be a big factor in Anytype's ability to support itself long term, it also opens more opportunity for people to start out with paying for a smaller plan. It's like a double edge sword. I'm glad that Anytype is looking into improving this.

**Comment by MaleicAcid** [comment]
+1 really need this. Hope to pay only for the external storage that exceeds the free tier, rather than purchasing a storage package with a specific capacity.

**Comment by eleazar7** [comment]
Yes, please also add some kind of option for a 1 time payment - buying permanent storage with just a one time payment.

**Comment by robotflaqd** [comment]
> Yes, please also add some kind of option for a 1 time payment - buying permanent storage with just a one time payment.

Would be nice, but only if it's sustainable long term; otherwise, no. Perhaps do it in the beginning for only a limited time.

**Comment by AlFirous** [comment]
> > Yes, please also add some kind of option for a 1 time payment - buying permanent storage with just a one time payment.
> 
> Would be nice, but only if it's sustainable long term; otherwise, no. Perhaps do it in the beginning for only a limited time.

Agree, limited time of lifetime storage is good for marketing side, otherwise it doesn't make sense. I've seen a lot of lifetime products sunset or change because it's not sustainable.

**Comment by SyndicatedPillbug** [comment]
Lifetime storage options would actually be fine with something like this if they meet the right size limits. For instance, sure, a lot of people might take a 100G lifetime package and live with that. But look at companies like pCloud that offer very large and expensive permanent storage options. They still do the bulk of their business with subscriptions, or subscriptions tacked onto permanent accounts.

So I could see account upgrades being a thing. After all, AnyType can store everything, so its data space requirements could become huge. 100G seems like a lot, 1TB seems like a lot, but those can actually fill up fast when you're using AnyType to store all your data.

**Comment by eleazar7** [comment]
True, all good points. I was thinking about this because Ardrive.io offers this for storage. 
I do see how this would not help fund the work or updates or new features. that does require some kind of investment.
maybe somewhere in between permanent storage a time fee and subscriptions to use the network. At the latest, your date would NOT get deleted if something were to happen and payment could not be made.
kinda like an archived idea.

**Comment by robotflaqd** [comment]
Maybe even the ability to purchase only a .any username. This is so that people using local only/self hosting can still benefit from having a .any network name when they don't need Anysync network space.

**Comment by MaleicAcid** [comment]
Can someone share the progress or timeline? I've been waiting for more than half a year and accumulated a lot of unsynced content. Can't wait to subscribe to the new membership program.

**Comment by trymeouteh** [comment]
https://community.anytype.io/t/more-plan-options/28472

Please consider making plans that is cheaper than the Basic plan with 128gb of storage or less (such as 64gb of storage space) with no unique names and no web publishing storage.

**Comment by eleazar7** [comment]
It would be great to have a basic plan that would at least give you permanent storage with no unique names and no web publishing storage.

Would love to connect to services like - internxt.com or Ardrive.io
even if its just a backup option. 

They offer Permanent storage for a 1 time payment. 

Now i am not saying that everything that Anytype offers should be under this kind of plan. There are other things that can be charged for as an ongoing service. I think this hits the best of both options of subscription and one time payment. 

this would bring more security our precious data. This also aligns with Anytypes core principle of data sovereignty.

**Comment by eleazar7** [comment]
- just found another one called "https://autonomi.com"
this is what i am talking about.
